products/messaging/frontend/Campaigns/hogflows/HogFlowEditor.tsx

```diff
@@ -64,13 +65,15 @@ function HogFlowEditorContent(): JSX.Element {
                 colorMode={isDarkModeOn ? 'dark' : 'light'}
                 onPaneClick={() => setSelectedNodeId(null)}
             >
-                <Controls showInteractive={false} />
-
                 <Background gap={36} variant={BackgroundVariant.Dots} />
 
+                <Controls showInteractive={false} />
+
                 <HogFlowEditorToolbar />
 
                 <HogFlowEditorDetailsPanel />
+
+                <HogFlowEditorTestPanel />
```

Comment:
Personally I think we should have one single panel - a right hand one for building and testing, and a left hand one for overall info about the campaign. For now its fine though